app.js

```diff
@@ -1,14 +1,17 @@
 const {
     categoryControllers: { getCategories },
-    reviewControllers: { getReviewById }
+    reviewControllers: { getReviewById, postReview }
 } = require("./controllers/index");
 
 const express = require("express");
 const app = express();
 
+app.use(express.json())
+
 app.get("/api/categories", getCategories);
 
 app.get("/api/reviews/:review_id", getReviewById)
+app.post("/api/reviews/:review_id", postReview)
```

Comment:
This is a patch request. You will have a ticket to post an actual review (I think), but for now we are just patching the votes of the review.